x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/index.tsx

```diff
@@ -100,9 +100,13 @@ const CreateRulePageComponent: React.FC = () => {
   const [activeStep, setActiveStep] = useState<RuleStep>(RuleStep.defineRule);
   const getNextStep = (step: RuleStep): RuleStep | undefined =>
     ruleStepsOrder[ruleStepsOrder.indexOf(step) + 1];
+  // @ts-expect-error forwardRef
```

Comment:
Is there anything we can do here to resolve this? I see other usages of `useRef` with different `Eui*` components that don't have this error, but seems `EuiAccordion` surfaces this? the other components (without error) are either a `class` or `interface` it looks like, so this is a result of `EuiAccordion` being exported as a `const` and so messes with type inference in `useRef` then?

<img width="1289" alt="image" src="https://user-images.githubusercontent.com/2946766/174664332-b9f38b2d-5454-4a54-98bb-cf9d233c8d52.png">

Reply:
Yeah thanks, I meant to take another pass at this. I think we'll need to export the underlying class so that it's accessible here. That'll  need to happen in a subsequent release, so I'll update the comment with an link to an EUI issue. We'll then resolve the `@ts-expect-error` in a future upgrade.

Reply:
Comment updated in [8bc8463](https://github.com/elastic/kibana/pull/133927/commits/8bc84638b206b573aaf5dc3506bae923c8b2fbfb)